src/token-tracker-link.ts

```diff
@@ -1,16 +1,23 @@
+import { addPathToUrl } from './helpers';
 import prefixForChain from './prefix-for-chain';
-import prefixForNetwork from './prefix-for-network'
+import prefixForNetwork from './prefix-for-network';
 
 export function createTokenTrackerLink(
   tokenAddress: string,
   network: string,
   holderAddress?: string,
+  customNetworkUrl?: string,
 ): string {
+  if(customNetworkUrl?.length){
+    const parsedUrl = addPathToUrl(customNetworkUrl, 'token', tokenAddress)
+    return parsedUrl;
+  } else {
   const prefix = prefixForNetwork(network)
   return prefix !== null ? 
       `https://${prefix}etherscan.io/token/${tokenAddress}${ 
         holderAddress ? `?a=${ holderAddress }` : '' }`
     : '';
+  }
```

Comment:
Oh, another indentation issue here:
```suggestion
  if (customNetworkUrl?.length){
    const parsedUrl = addPathToUrl(customNetworkUrl, 'token', tokenAddress)
    return parsedUrl
  } else {
    const prefix = prefixForNetwork(network)
    return prefix !== null ? 
      `https://${prefix}etherscan.io/token/${tokenAddress}${ 
        holderAddress ? `?a=${ holderAddress }` : '' }`
      : ''
  }
```